### PR TITLE
Makes changes for lint warnings issue #507 - getMembers.test.js

### PR DESCRIPTION
**Pull request by ethanstrominger**

### CODE REVIEW

File: src/test/utils/getMembers.test.js [hunk 1]
```diff
@@ -18,39 +18,51 @@ import {
 
 // getEveryMember(array, fn) = array.filter(fn)
 test('getEveryMember()', () => {
-  expect(getEveryMember([ 0, 1, 0 ], (val) => {return val === 0;})).toHaveLength(2);
+  expect(getEveryMember([ 0, 1, 0 ], (val) => {
+    return val === 0;}
```
Comment (wacii) [inline src/test/utils/getMembers.test.js]: Curly brace on next line.

File: src/test/utils/getMembers.test.js [hunk 2]
```diff
@@ -18,39 +18,51 @@ import {
...
+  });
   const allMembers = headOrSpouse.concat(notHeadOrSpouse);
 
   test('isHeadOrSpouse()', () => {
-    headOrSpouse.forEach((member) => {return expect(isHeadOrSpouse(member)).toBe(true);});
-    notHeadOrSpouse.forEach((member) =>
-    {return expect(isHeadOrSpouse(member)).toBe(false);}
-    );
+    headOrSpouse.forEach((member) => {
+      return expect(isHeadOrSpouse(member)).toBe(true);
```
Comment (wacii) [inline src/test/utils/getMembers.test.js]: Doesn't need to return.

File: src/test/utils/getMembers.test.js [hunk 3]
```diff
@@ -18,39 +18,51 @@ import {
...
+    });
+    notHeadOrSpouse.forEach((member) => {
+      return expect(isHeadOrSpouse(member)).toBe(false);
```
Comment (wacii) [inline src/test/utils/getMembers.test.js]: Also doesn't need to return.

File: src/test/utils/getMembers.test.js [hunk 4]
```diff
@@ -18,39 +18,51 @@ import {
...
-    );
-    notHeadOrSpouse.forEach((member) =>
-    {return expect(isNotHeadOrSpouse(member)).toBe(true);}
-    );
+    headOrSpouse.forEach((member) => {
+      return expect(isNotHeadOrSpouse(member)).toBe(false);
+    });
+    notHeadOrSpouse.forEach((member) => {
+      return expect(isNotHeadOrSpouse(member)).toBe(true);
+    });
```
Comment (wacii) [inline src/test/utils/getMembers.test.js]: Doesn't need to return.

File: src/test/utils/getMembers.test.js [hunk 5]
```diff
@@ -69,8 +81,12 @@ describe('dependents', () => {
...
 
   test('isDependent()', () => {
-    dependents.forEach((member) => {return expect(isDependent(member)).toBe(true);});
-    notDependents.forEach((member) => {return expect(isDependent(member)).toBe(false);});
+    dependents.forEach((member) => {
+      return expect(isDependent(member)).toBe(true);
+    });
+    notDependents.forEach((member) => {
+      return expect(isDependent(member)).toBe(false);
+    });
```
Comment (wacii) [inline src/test/utils/getMembers.test.js]: Doesn't need to return.

File: src/test/utils/getMembers.test.js [hunk 6]
```diff
@@ -104,13 +120,21 @@ describe('disabled', () => {
...
 
   test('isUnder13()', () => {
-    under13.forEach((member) => {return expect(isUnder13(member)).toBe(true);});
-    over12.forEach((member) => {return expect(isUnder13(member)).toBe(false);});
+    under13.forEach((member) => {
+      return expect(isUnder13(member)).toBe(true);
+    });
+    over12.forEach((member) => {
+      return expect(isUnder13(member)).toBe(false);
+    });
```
Comment (wacii) [inline src/test/utils/getMembers.test.js]: Doesn't need to return.

**Comment by knod** [comment]
@wacii: Ethan isn't editing this to change the code, just to fix the linting warnings.

File: src/test/utils/getMembers.test.js (hunk 1 above)
Comment (knod) [reply src/test/utils/getMembers.test.js]: Yeah, this is the one that needs to be changed to finish with the linting stuff.

**Comment by wacii** [comment]
@knod Sure, but considering the return statements were introduced by ESLint, I thought it might be relevant.

**Comment by knod** [comment]
@wacii: From what I can tell, eslint is doing it's job correctly, it just leaves us with slightly excessive, but completely functional, code. I'm not sure fixing eslint's corrections is a task our human-powered lint-fixers can tackle at this time. I'll bring it up in the 'tech' channel for a discussion.

**Pull request merged**